conda/environments/cuml_dev_cuda11.0.yml

```diff
@@ -6,14 +6,14 @@ channels:
 - conda-forge
 dependencies:
 - cudatoolkit=11.0
-- rapids-build-env=21.08.*
-- rapids-notebook-env=21.08.*
-- rapids-doc-env=21.08.*
-- cudf=21.08.*
-- rmm=21.08.*
-- libcumlprims=21.08.*
-- dask-cudf=21.08.*
-- dask-cuda=21.08.*
+- rapids-build-env=21.10.*
+- rapids-notebook-env=21.10.*
+- rapids-doc-env=21.10.*
+- cudf=21.10.*
+- rmm=21.10.*
+- libcumlprims=21.10.*
+- dask-cudf=21.10.*
+- dask-cuda=21.10.*
```

Comment:
This 21.10 changes were probably accidentally commited due to starting from a branch-21.10?